packages/api/tests/lambdas/test-publish-reports.js

```diff
@@ -379,7 +379,7 @@ test.serial('failure describing step function in handleGranuleMessages does not
   }
 });
 
-test.serial('lambda publishes correct granules from payload.granules to SNS topic', async (t) => {
+test.serial.only('lambda publishes correct granules from payload.granules to SNS topic', async (t) => {
```

Comment:
You need to remove `.only` 😄

Reply:
Fixed in 59845ed6b. I really need to add a pre-commit hook for that.